app/components/Issues/Detail/IssueDetailBody.jsx

```diff
@@ -79,7 +80,13 @@ const IssueDetailBody = ({
     <Fragment>
       <PostingInfoWrapper>
         <div>
-          Opened by <UsernameLink to={route}>{username}</UsernameLink> on&nbsp;
+          Opened by{' '}
```

Comment:
What are your thoughts on just using one styled-component for this? Perhaps we can pass in a variable that would determine in `pointer-events` should be `none` or `inherit`.
```
const isUserDeleted = username === '[deleted]';
...

<div>
          Opened by <UsernameLink isUserDeleted={isUserDeleted} to={route}>{username}</UsernameLink> on&nbsp;
          {moment(date)
            .utc()
            .format('M/D/YYYY')}
 </div>
```

Reply:
Yeah I think that's way better, I didn't know pointer-events was a thing! Thank you, and I'll be sure to do that right away.